src/sentry/conf/server.py

```diff
@@ -3071,6 +3071,14 @@ def build_cdc_postgres_init_db_volume(settings: Any) -> dict[str, dict[str, str]
             "only_if": settings.SENTRY_USE_SPOTLIGHT,
         }
     ),
+    "peanutbutter": lambda settings, options: (
+        {
+            "image": "us.gcr.io/sentryio/peanutbutter:latest",
+            "environment": {},
+            "ports": {"50051/tcp": 50051},
```

Comment:
this is inside the ephemeral port range ~32k - 65k and so it will cause conflicts with randomly running connections

please pick a lower fixed port number